### PR TITLE
fix(client): Content misalignment on the profile page

### LINKED ISSUE #51955: Content misalignment on the profile page
### Describe the Issue

This issue was reported on Discord: https://discord.com/channels/692816967895220344/693145545878929499/1162395054237831301.

The certification links are shifted to the left instead of being centered:

Screenshot 2023-10-13 at 21 56 47

### System

N/A

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by RickDeb2004**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #51955
image

### CODE REVIEW

File: client/src/components/profile/components/certifications.tsx [hunk 1]
```diff
@@ -49,7 +49,7 @@ function CertButton({ username, cert }: CertButtonProps): JSX.Element {
   return (
     <>
       <Row>
-        <Col className='certifications' sm={10} smPush={1}>
+        <Col className='certifications text-center' sm={10} smPush={1}>
```
Comment (moT01) [inline client/src/components/profile/components/certifications.tsx:52]:
```suggestion
        <Col className='certifications' sm={10} smOffest={1}>
```

**Review by moT01 — CHANGES_REQUESTED** [review]
This doesn't seem to fix the issue as-is @RickDeb2004. Just to clarify, your screen shot is on the [curriculum map](https://www.freecodecamp.org/learn) - the issue is on [a profile page](https://www.freecodecamp.org/mot01). I loaded up this PR and the buttons were still misaligned. I did a little digging and found that the problem was introduced on [this PR.](https://github.com/freeCodeCamp/freeCodeCamp/pull/51322) Hey @Sboonny, was this something that was just missed on that PR? or is something not working as expected?

Can you update the PR description so that it closes the associated issue @RickDeb2004. Also, make sure you do all the tasks in the PR checklist.

**Comment by RickDeb2004** [comment]
> This doesn't seem to fix the issue as-is @RickDeb2004. Just to clarify, your screen shot is on the [curriculum map](https://www.freecodecamp.org/learn) - the issue is on [a profile page](https://www.freecodecamp.org/mot01). I loaded up this PR and the buttons were still misaligned. I did a little digging and found that the problem was introduced on [this PR.](https://github.com/freeCodeCamp/freeCodeCamp/pull/51322) Hey @Sboonny, was this something that was just missed on that PR? or is something not working as expected?
> 
> Can you update the PR description so that it closes the associated issue @RickDeb2004. Also, make sure you do all the tasks in the PR checklist.

not able to see my changes as I have done some changes in profile directory.

**Comment by RickDeb2004** [comment]
> Checklist:
> 
> * [ done] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
> * [done ] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
> * [ done] My pull request targets the `main` branch of freeCodeCamp.
> * [ done] I have tested these changes either locally on my machine, or GitPod.
> 
> Closes #XXXXX image

close: #51955

**Review by huyenltnguyen — CHANGES_REQUESTED** [review]

**Comment by Sboonny** [comment]
> Hey @Sboonny, was this something that was just missed on that PR? or is something not working as expected?

Hi @moT01, although I remember fixing this by adding `relative` className, it isn't there 🤔, so it is something missed that should have been accounted for in that PR. 

if this is merged with the switching to `smOffset`, please delete the logic around `smPush` as it is no longer necessary

**Review by huyenltnguyen — APPROVED** [review]

**Review by moT01 — APPROVED** [review]
LGTM @RickDeb2004 🎉 Congratulations on your first contribution to the freeCodeCamp repo 🎉

**Pull request merged**